specs/DesignDiscussions/ImagevColumn.md

```diff
@@ -0,0 +1,65 @@
+# Consistency spec: Image size vs Column size
+
+## Table of Contents
+- [Overview](#overview)
+    - [Current Problem](#current-problem)
+    - [What's happening behind the scenes?](#whats-happening-behind-the-scenes)
+    - [Proposed Solutions](#proposed-solutions)
+        - [Honoring Image Size](#honoring-image-size)
+
+## Overview
+
+### Current problem
+
+As part of the solution to the Column width behaviour it was thought that images could be used as a reference to provide the minimum size needed to render an `auto` width column. This asseveration is not correct, as some of our scenario cards don't follow that principle, take for example the [FlightDetails card](), in this card we have an image whose `width` is `1000px`, if we were to honour the measurements given by the card author we would get a card where this long image occupies most of the width of the column set as can be seen in the views generated by the iOS and the Android renderers.
```

Comment:
Thanks for trying to add context here. But can you also link to the "Column width behavior" issue that you are referring to here for clarification? Thanks

Reply:
Is it https://github.com/microsoft/AdaptiveCards/pull/4722 ?